test/test_helper.rb

```diff
@@ -10,9 +10,17 @@
 #
 ################################################################################
 
-# Coveralls.wear! must occur before any of your application code is required
+# Code to allow both local and coveralls coverage.  From:
+# https://coveralls.zendesk.com/hc/en-us/articles/201769485-Ruby-Rails
+require "simplecov"
 require "coveralls"
-Coveralls.wear!("rails")
+
+# Coveralls.wear!("rails")
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
```

Comment:
This line caused a deprecation notice when I ran rake.  I don't understand the notice (or how to fix it).
```ruby
$ rake
/vagrant/mushroom-observer/test/test_helper.rb:19:in `<top (required)>': [DEPRECATION] ::[] is deprecated. Use ::new instead.
```